deploy/040_HOPRBoost-minting.ts

```diff
@@ -12,8 +12,10 @@ const main: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // We obtain the just deployed HoprBoost contract address based on the tx
   const boostContract = await ethers.getContractAt("HoprBoost", HoprBoost.address) as HoprBoost;
 
-  // We mint a single NFT to alice, and pass the proper values
-  const mintTx = await boostContract
+  console.log(`Private key of target account used for minted NFTs: ${PRIVATE_KEY_ALICE}`);
+
```

Comment:
You are right that `Signer` type cannot export private key. A walk around is like below

```TypeScript
// default mnenomic set in hardhat https://hardhat.org/hardhat-network/reference/#initial-state
  const mnemonic = "test test test test test test test test test test test junk"
  const accounts = await ethers.getSigners();
  const aliceAccountIndex = accounts.findIndex(account => account.address.toLowerCase() === alice.toLowerCase());
  const aliceAccount = ethers.Wallet.fromMnemonic(mnemonic, `m/44'/60'/0'/0/${aliceAccountIndex}`)
  assert(aliceAccount.address.toLowerCase() === alice.toLowerCase(), "cannot find alice in default accounts")
  console.log(`Private key of alice is ${aliceAccount.privateKey}`)
```

This block can be added under L.8 `const { admin, alice } = await getNamedAccounts();`

Reply:
We agreed to keep it as is for now.